test/stop-operator.sh

```diff
@@ -45,32 +48,74 @@ function delete_olm_operator() {
 
 function delete_operator() {
   DEPLOY_DIRECTORY="${REPO_DIRECTORY}/deploy/operator"
-  deploy="${DEPLOYMENT_DIRECTORY}/pmem-csi-operator.yaml"
+  deploy="${DEPLOY_DIRECTORY}/pmem-csi-operator.yaml"
+
+  tmpdir=$(mktemp -d)
+  trap "rm -rf $tmpdir" SIGTERM SIGINT EXIT
+
+  cat > ${tmpdir}/pmem-csi-operator.yaml <<EOF
+$(cat "${deploy}")
+EOF
```

Comment:
How is this different from `cp "${deploy}" ${tmpdir}/pmem-csi-operator.yaml`?

Reply:
Introducing and using the `deploy` variable only once makes this function harder to follow.

Reply:
Removed the unneeded variables.